Content.Shared/Actions/Events/FabricateActionEvent.cs

```diff
@@ -0,0 +1,9 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Actions.Events;
+
+public sealed partial class FabricateActionEvent : InstantActionEvent
```

Comment:
```suggestion
public sealed class FabricateActionEvent : InstantActionEvent
```

Reply:
It's a DataDefinition so it needs to be partial

Reply:
Fuck, I accidentally applied the suggestion too... well, fuck me.

Reply:
webed**it**:tm: